packages/hub/prisma/schema.prisma

```diff
@@ -356,8 +360,8 @@ enum WalletOrdersStatusEnum {
   @@map("wallet_orders_status_enum")
 }
 
-enum ScheduledPaymentAttemptsStatusEnum {
-  in_progress
+enum ScheduledPaymentAttemptStatusEnum {
+  inProgress @map("in_progress")
```

Comment:
There seem to be a number of string versions of "inProgress" in this code still. Are we sure everything is lined up right with the formatting of this value?

Reply:
This just tells prisma that inProgress is maps to `in_progress` when executing the query. So it's expected the code is using `inProgress`. 

Proof (note that `in_progress` is used in SQL while `inProgress` is used in the code) :

```
Hub > service.fetchScheduledPayments()
Hub > {
  timestamp: 2022-09-02T09:42:11.516Z,
  query: 'SELECT "public"."scheduled_payments"."id", "public"."scheduled_payments"."sender_safe_address", "public"."scheduled_payments"."module_address", "public"."scheduled_payments"."token_address", "public"."scheduled_payments"."amount", "public"."scheduled_payments"."payee_address", "public"."scheduled_payments"."execution_gas_estimation", "public"."scheduled_payments"."max_gas_price", "public"."scheduled_payments"."fee_fixed_usd", "public"."scheduled_payments"."fee_percentage", "public"."scheduled_payments"."salt", "public"."scheduled_payments"."pay_at", "public"."scheduled_payments"."recurring_day_of_month", "public"."scheduled_payments"."recurring_until", "public"."scheduled_payments"."valid_for_days", "public"."scheduled_payments"."sp_hash", "public"."scheduled_payments"."chain_id", "public"."scheduled_payments"."creation_transaction_hash", "public"."scheduled_payments"."creation_block_number", "public"."scheduled_payments"."cancelation_transaction_hash", "public"."scheduled_payments"."cancelation_block_number", "public"."scheduled_payments"."created_at", "public"."scheduled_payments"."updated_at", "public"."scheduled_payments"."canceled_at" FROM "public"."scheduled_payments" WHERE ("public"."scheduled_payments"."canceled_at" IS NULL AND "public"."scheduled_payments"."creation_block_number" > $1 AND "public"."scheduled_payments"."pay_at" > $2 AND "public"."scheduled_payments"."pay_at" <= $3 AND (("public"."scheduled_payments"."recurring_day_of_month" IS NULL AND ("public"."scheduled_payments"."id") NOT IN (SELECT "t0"."id" FROM "public"."scheduled_payments" AS "t0" INNER JOIN "public"."scheduled_payment_attempts" AS "j0" ON ("j0"."scheduled_payment_id") = ("t0"."id") WHERE (("j0"."status" = $4 OR "j0"."status" = $5) AND "t0"."id" IS NOT NULL))) OR ("public"."scheduled_payments"."recurring_day_of_month" > $6 AND "public"."scheduled_payments"."recurring_until" >= $7 AND ("public"."scheduled_payments"."id") NOT IN (SELECT "t0"."id" FROM "public"."scheduled_payments" AS "t0" INNER JOIN "public"."scheduled_payment_attempts" AS "j0" ON ("j0"."scheduled_payment_id") = ("t0"."id") WHERE ("j0"."status" = $8 AND "t0"."id" IS NOT NULL))))) OFFSET $9',
  params: '[0,2022-08-22 22:00:00 UTC,2022-09-02 07:42:11 UTC,"succeeded","in_progress",0,2022-09-02 07:42:11 UTC,"in_progress",0]',
  duration: 5,
  target: 'quaint::connector::metrics'
}
```